pkgs/development/tools/misc/terramate/default.nix

```diff
@@ -6,24 +6,24 @@
 
 buildGoModule rec {
   pname = "terramate";
-  version = "0.8.4";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
     owner = "terramate-io";
     repo = "terramate";
     rev = "v${version}";
-    hash = "sha256-jcmOS81iPzy1ul0Cj/SiJk84AUIq7mLI+CmspuPit+o=";
+    hash = "sha256-052pw091okx564tzN3Ookw2xZmW12svI49Ll5MIWMu0=";
   };
 
-  vendorHash = "sha256-Na2XDPSwgwWTQrweslAtSOh2+B/ZFaPIdy8ssAFWkGs=";
+  vendorHash = "sha256-lJL8THy0wXrKTSPdnBxollgZkaHMhvw3MbuiacyR6GE=";
 
   # required for version info
   nativeBuildInputs = [ git ];
 
   ldflags = [ "-extldflags" "-static" ];
 
   # Disable failing E2E tests preventing the package from building
-  excludedPackages = [ "./cmd/terramate/e2etests/cloud" "./cmd/terramate/e2etests/core" ];
+  excludedPackages = [ "./e2etests/cloud" "./e2etests/core" ];
```

Comment:
Moved in: https://github.com/terramate-io/terramate/commit/2349f49c998dcf8b1270223a8c44bbaabdff0d23